src/components/Button.stories.tsx

```diff
@@ -130,3 +130,7 @@ export function Buttons(args: ButtonProps) {
     </div>
   );
 }
+
+export function ButtonWithTooltip() {
+  return <Button disabled={true} disabledReason="You cannot currently perform this operation." label="Upload" />;
```

Comment:
**Thoughts:** On having the disabled prop take in a `boolean` or a string for the message? Just thinking that disabled and disabledReason would always be in sync.